SPECS/authselect/authselect.spec

```diff
@@ -0,0 +1,371 @@
+# Do not terminate build if language files are empty.
+%define _empty_manifest_terminate_build 0
+
+Name:           authselect
```

Comment:
`> Disable translation in authselect as it requires multiple packages to be moved to core`

I think we should confirm this with @adobrzhansky - if this is a breaking change for their usecase.

Reply:
I see no issues that may be related with that as long as nss package is working